src/middleware/navigation/sign.out.base.url.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Middleware to generate the sign-out baseUrl for the respective journeys
+ * Assumes "res.locals.journey" has been set from previous middleware executions
+ */
+import { NextFunction, Request, Response } from "express";
+
+import * as config from "../../config";
+import { logger } from "../../utils/logger";
+import {
+  isRegistrationJourney,
+  getUrlWithParamsToPath,
+  transactionIdAndSubmissionIdExistInRequest
+} from "../../utils/url";
+
+export const generateSignOutBaseUrl = (req: Request, res: Response, next: NextFunction) => {
+  logger.info("setting sign-our-base-url");
```

Comment:
can you use logger.infoRequest(req, "setting sign-our-base-url"); instead of logger.info